extensions/msal-node-extensions/src/lock/CrossPlatformLock.ts

```diff
@@ -57,7 +57,7 @@ export class CrossPlatformLock {
         try {
             // delete lock file
```

Comment:
Should the comment change? As you are `unlinking` and not `deleting`?

Reply:
We are deleting the file. https://nodejs.org/api/fs.html#fs_fs_unlink_path_callback